docs/source/upcoming_release_notes/1283-Adding_base_class_for_lakeshore_336..rst

```diff
@@ -0,0 +1,32 @@
+1283 Adding base class for lakeshore 336.
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+-Base class for lakeshore 336
+-Class for Temperature Sensor
+-Class for Heater
+
+Device Updates
+--------------
+- N/A
```

Comment:
You should make sure your fork is up-to-date after this, recently we changed the template slightly. I think this is still good to merge.